src/pages/editProfile.tsx

```diff
@@ -153,101 +204,107 @@ export default function Home() {
-              style={
-                !isImageUpload ? { opacity: "0.5" } : { objectFit: "cover" }
-              }
-            />
-          )}
-          <IconButton
-            sx={overlayIcon}
-            aria-label="upload picture"
-            component="label"
-          >
-            <input
-              onChange={handleImageChange}
-              hidden
-              accept="image/*"
-              type="file"
-            />
-            <CameraAltIcon sx={{ fontSize: "100px" }} />
-          </IconButton>
-        </Avatar>
-        {showImageUploadError.err && (
-          <Typography align="center" variant="body1" color="error">
-            {showImageUploadError.msg}
-          </Typography>
-        )}
-
-        <Box style={editInfoContainer}>
-          <CommonTextField
-            header={editProfileHeader.displayName}
-            value={displayName}
-            handleValueChange={handleDisplayNameChange}
-            char_limit={CHAR_LIMIT.MAX_DISPLAY_NAME}
-            isErr={isPressSubmit && displayNameErr.err}
-            errMsg={displayNameErr.msg}
-          />
-
-          <CommonTextField
-            header={editProfileHeader.description}
-            value={description}
-            handleValueChange={handleDescChange}
-            isMultiLine={true}
-            char_limit={CHAR_LIMIT.MAX_DESCRIPTION}
-            isErr={isPressSubmit && descriptionErr.err}
-            errMsg={descriptionErr.msg}
-          />
-
-          <Box style={{ width: "18vw" }}>
-            <CommonDropdown
-              header={editProfileHeader.gender}
-              value={gender}
-              handleValueChange={handleSelectChange}
-              items={Object.values(Gender)}
-            />
+      <Stack alignItems="center">
+        <Card sx={EditProfileStyle.Card}>
+          <Box display="flex">
+            <Box sx={{ flexGrow: 1 }}></Box>
+            <IconButton onClick={handleOpenModal}>
+              <CloseIcon fontSize="large" color="secondary" />
+            </IconButton>
           </Box>
-        </Box>
-
-        <Button variant="contained" onClick={editProfileBtnOnClick}>
-          SAVE
-        </Button>
-        <Typography variant="body1" color="warning.main">
-          ท่านต้องกด SAVE เพื่อบันทึกข้อมูลใหม่
-        </Typography>
+          <Stack
+            spacing={2}
+            sx={{ marginTop: "-40px" }}
+            alignItems="center"
+            justifyContent="center"
+          >
+            <IconButton
+              onClick={handleCancelImageChip}
+              color="secondary"
+              style={{
+                padding: 0,
+                backgroundColor: "black",
+                position: "relative",
+                top: "70px",
+                left: "80px",
+                zIndex: "1",
+              }}
+            >
+              <CloseIcon color={showImageUploadError.err ? "error" : "primary"} fontSize="medium" />
+            </IconButton>
```

Comment:
At first, I don't have a profile pic so I add and delete for several time before press save button, it works just fine. But when I edit it again the close icon on the image doesn't remove my profile pic.

Reply:
this you have this warning? It appears when I refresh the page (also if I upload the new pic overwriting the present one it doesn't have any problem tho)
<img width="493" alt="image" src="https://user-images.githubusercontent.com/62459518/226865391-e07e0d81-b039-477a-b348-9483531ec096.png">

Reply:
As I said, I didn't change any logic and this seem like to be error on displaying error. Could you recap the restriction of error message on profile image?, so that I can clean it up.

Reply:
Solved the latter but the former is wating for the recap.